strongbox-web-core/src/main/java/org/carlspring/strongbox/config/WebConfig.java

```diff
@@ -138,7 +138,7 @@ protected String createMessage(HttpServletRequest request,
 
         };
         result.setIncludeQueryString(true);
-        result.setIncludeHeaders(true);
```

Comment:
Not sure if this is a good idea since this will remove the headers. We should probably add `beforeRequest` and `afterRequest` and redact the commonly known sensitive headers (i.e. `Authentication: Bearer TOKEN` => `Authentication: Bearer [credentials redacted]`). 

@sbespalov thoughts?

Reply:
Added `result.setHeaderPredicate(s -> !(s.equals("Authorization") || s.equals("Cookie")));` which mask the authorization and cookies in header.

Header shown in log  -> `[method=GET;GET /api/account, client=0:0:0:0:0:0:0:1, user=admin, headers=[Authorization:"masked", Cookie:"masked", Accept:"application/json", X-Requested-With:"XMLHttpRequest", User-Agent:"Mozilla/5.0 (X11; Linux x86_64; rv:76.0) Gecko/20100101 Firefox/76.0", Connection:"keep-alive", Referer:"http://localhost:48080/my-account", Host:"localhost:48080", Accept-Language:"en-US,en;q=0.5", Accept-Encoding:"gzip, deflate", DNT:"1"]]`